_data/authors.yml

```diff
@@ -0,0 +1,26 @@
+rohitrox:
+  username: rohitrox
+  name: Rohit Joshi
+  location: Kathmandu, Nepal
+  twitter-username: "rohitroxxxxx"
+  github-username: "RohitRox"
+  bio: "Proud to be a kapil fan nd a gautam rode fan met gautam rode on 5th july'2014 and he followed me on 7th
+  july'2014(feeling wonderful) a huge animal lover... :)"
+  photo: https://pbs.twimg.com/profile_images/531128640239448064/us7VkVAQ_400x400.jpeg
+zoras:
+  username: zoras
+  name: Saroj Maharjan
+  location: Kathmandu, Nepal
+  twitter-username: "zoras"
+  github-username: "zoras"
+  bio: "Web 🕸️ #Developer, Mentor, Freelancer, @Ruby_Nepal Activist, Meetup Organizer, Available for #Ruby &
+   #Rails Consulting. 💕 @Arsenal 🇳🇵 ✉️ http://scr.im/zoras"
+  photo: https://pbs.twimg.com/profile_images/762590475593199616/dl4L9IW9_400x400.jpg
+josisusan:
+  username: josisusan
+  name: Sushan Joshi
```

Comment:
Susan Joshi

Reply:
@josisusan  dai, Bio and photo link is copied from twitter, Make sure that is yours :)